QUEST_LV_0100.tsv

```diff
@@ -4425,23 +4425,23 @@ QUEST_LV_0100_20150317_004424	Obtain %s by defeating Shnayim
 QUEST_LV_0100_20150317_004425	Defeat Shnayim
 QUEST_LV_0100_20150317_004426	Talk to Vio
 QUEST_LV_0100_20150317_004427	The commander, Vio is looking for his men. Talk to Vio.
-QUEST_LV_0100_20150317_004428	Find Mercenary Lint
-QUEST_LV_0100_20150317_004429	Vio want you to find his men who ran away somewhere. Let's first go to Apnis Camp where Lint is likely to be at.
-QUEST_LV_0100_20150317_004430	Talk to Lint
-QUEST_LV_0100_20150317_004431	The man who is leaning on the tree and taking a rest seems to be Lint. Talk to him since he seems badly hurt.
+QUEST_LV_0100_20150317_004428	$Find Mercenary Lynt
+QUEST_LV_0100_20150317_004429	Vio want you to find his men who ran away somewhere. Let's first go to Apnis Camp where Lynt is likely to be at.
+QUEST_LV_0100_20150317_004430	$Talk to Lynt
+QUEST_LV_0100_20150317_004431	The man who is leaning on the tree and taking a rest seems to be Lynt. Talk to him since he seems badly hurt.
 QUEST_LV_0100_20150317_004432	Retrieve the supplies which Wendigo stole
-QUEST_LV_0100_20150317_004433	Lint told you that he was chasing after Wendigo for stolen supplies instead of running away from the monsters. Anyways, defeat Wendigo and retrieve the stolen supplies for Lint.
-QUEST_LV_0100_20150317_004434	Give the supplies to Lint
-QUEST_LV_0100_20150317_004435	{memo X}You have retrieved all the supplies that were stolen from Wendigo. Give them to Lint.
+QUEST_LV_0100_20150317_004433	$Mercenary Lynt told you that he was chasing after Wendigo for stolen supplies instead of running away from the monsters. Anyways, defeat Wendigo and retrieve the stolen supplies for Lint.
```

Comment:
Remove "Anyways, " here. It doesn't really fit.

Also forgot to change the second Lint to Lynt.